README.md

```diff
@@ -64,3 +64,26 @@ To install the development version:
 ```
 conda install -c usgs-astrogeology/label/dev plio
 ```
+
+Building documentation
+======================
+
+Create the `plio` conda environment:
+
+```
+conda env create -f environment.yml
```

Comment:
Sorry, premature approve, this needs to be environment_docs.yml

Reply:
I actually updated the `environment.yml` file to support docs build (just added `sphinx` dep), should I revert that?

Reply:
is it enough to completely build the docs? Pretty sure you need all the docs deps to build it not just sphinx-build command. I manually installed sphinx earlier and ended up with that error from before.

Reply:
Oh hm, I tested it out by creating a new conda env with the updated yaml file and ran the `sphinx-build` command without issue. Also, I did have to pin the `sphinx` version to `6.2.1` so if you did a casual `conda install sphinx`, it may have installed a version (`7.0+`) that is known to cause the error you ran into.

Reply:
yeah that seemed to be the issue.